ansible/create-solaris-esxi.md

```diff
@@ -0,0 +1,88 @@
+# Instructions for creating Solaris machines in ESXi
+
+## Creating the new machine template
+
+1. Login to https://esxi.adoptopenjdk.net. @gdams and @sxa have credentials if needed.
+1. Click `Create / Register VM`.
+1. Click `Create a new virtual machine`.
+1. Select a name and guest OS:
+    - Name: Provide a hostname
+    - Compatibility: `ESXi 6.5 virtual machine`.
+    - Guest OS family: `Other`.
+    - Guest OS version: `Oracle Solaris 10 (64-bit)`
+1. Leave the storage as `datastore1`.
+1. Customize settings:
+    - CPU: Select `4` unless more is needed.
+    - Memory: Select `8GB` unless more is needed.
+    - Hard disk 1: Select a minimum of 120GB.
+    - Network Adapter 1: Ensure that `VM Network` is selected and the `Connect` box is ticked.
+    - CD/DVD Drive 1: Select `Datastore ISO file`, then select `sol-10-u11-ga-x86-dvd.iso`.
+1. Click `Finish`
+
+
+## First boot and Solaris Installation
+
+### System Identification
+
+1. Once the machine has been created, click on it and select the `Power on` option.
+1. You should then be able to click the console screen in the left corner which should display the `GNU GRUB` loader.
+1. `Oracle Solaris` should boot by default or you can hit the enter button.
+1. Type `1` and then enter (Oracle Solaris Interactive default)
+1. Select the keyboard layout (in my case `UK-English`) and then hit `F2`.
+1. Press Enter in the screen test shell.
+1. Select a language (in my case `0`), then hit Enter.
+1. Click `F2` for the next few screen using the default settings until you get to the hostname.
+1. Set the hostname to match what you set the machine name is ESXi to be and then hit `F2`.
+1. Set the IP address. The current block of IP's that we have is `147.75.85.208/29`.
```

Comment:
Suggest we make this a bit easier to find out which ones we have available - where to look etc. e.g. the two inventory files that we have. Perhaps also list the explicit range of addresses instead of just the `/29` mask.

Reply:
```suggestion
1. Set the IP address. The current block of IP's that we have is `147.75.85.208/29` (8 addresses) See the [inventory.yml](https://github.com/temurin-compliance/infrastructure/blob/master/ansible/inventory.yml) to find out which of those are already in use.
```